src/Microsoft.Health.Fhir.Shared.Core/Features/Search/Parameters/SearchParameterToTypeResolver.cs

```diff
@@ -326,15 +339,7 @@ private static ClassMapping GetMapping(Type type)
 
         private static ModelInspector GetModelInspector()
         {
-            string methodName = "GetStructureDefinitionSummaryProvider";
-            MethodInfo modelInspectorMethod = typeof(ModelInfo).GetMethod(methodName, BindingFlags.NonPublic | BindingFlags.Static);
-
-            if (modelInspectorMethod == null)
-            {
-                throw new MissingMethodException(nameof(ModelInfo), methodName);
-            }
-
-            return (ModelInspector)modelInspectorMethod.Invoke(null, null);
+            return ModelInspector.ForAssembly(typeof(ModelInfo).GetTypeInfo().Assembly);
```

Comment:
I guess this makes sense. Only downside is there will be 2 of these collections in memory if the base SDK is also still doing it somewhere

Reply:
Oh, this is why this reflection magic was here...